### PR TITLE
Restore postStartCommand.sh for backward compatibility

### PR DESCRIPTION
**Pull request by BjoernAtBosch**

### CODE REVIEW

File: setup/src/common/scripts/postStartCommand.sh [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations
+# under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Put your "post start" actions here:
```
Comment (erikbosch) [inline setup/src/common/scripts/postStartCommand.sh:16]: Should we possible remove this line as a "user" cannot just change these files as then the sync will complain that the files have been modified.

**Review by erikbosch — APPROVED** [review]
OK, does not really matter if comment is kept or not

File: setup/src/common/scripts/postStartCommand.sh (hunk 1 above)
Comment (BjoernAtBosch) [reply setup/src/common/scripts/postStartCommand.sh:16]: I guess, you're right. Removed it.

**Review by erikbosch — APPROVED** [review]
LGTM

**Pull request merged**